core/server/worker/src/main/java/alluxio/worker/netty/DataServerBlockReadHandler.java

```diff
@@ -113,7 +114,6 @@ public void close() {
       } catch (Exception e) {
         LOG.warn("Failed to unlock block {} with error {}.", mId, e.getMessage());
       }
-      mWorker.cleanupSession(mSessionId);
```

Comment:
Because we have already cleaned up in the "try" section which is essentially the same as cleanupSession here. Without this, I see lots of warning messages in the log.